package.json

```diff
@@ -91,14 +91,14 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
+    "jest": "jest",
     "lint:js": "eslint .",
     "lint:md": "remark . --quiet --frail",
     "lint": "npm-run-all --parallel lint:*",
     "pretest": "npm-run-all --serial lint flow",
     "release": "npmpub",
-    "test:jest": "jest",
-    "test": "npm-run-all --serial test:*",
-    "watch": "npm run test:jest -- --watch"
+    "test": "jest",
+    "watch": "npm run jest -- --watch"
```

Comment:
Can we have just `jest --watch` here please? I sometimes find the extra `--` unreliable when passing _further_ flags into the npm command e.g. `npm run watch -- logHeapSize`.